### DISCUSSION TITLE
I can't access my account

### DISCUSSION THREAD

**Discussion by Licayru** [community/community#127108] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello, I need help. I have an account that I can't access because the email address I used was the one my school gave me a couple of years ago, but that account is currently suspended. I have spoken to the school, but it is not possible to reinstate it.

When I try to log in to that GitHub account, I can't access it because a confirmation email is sent to that email address, but I can't validate it because it is suspended. Is there any way to recover my account? Or, if it is not possible, is there a way to delete that account?

I'm trying to contact GitHub support, but I'm not getting a response. I don't know how to proceed.

**Comment by ebndev** [comment]
Hi @Licayru,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
